front/sammendrag.tex

```diff
@@ -3,24 +3,25 @@
 \chapter{Sammendrag}
 \label{chap:sammendrag}
 
-Innvevde datasystemer blir gradvis en større del av hverdagen vår.
-Disse datamaskinene må stadig tilpasse seg til nye domener, slik som f.eks. tingenes internett.
-Det er mange begrensninger i disse systemene som setter krav til utviklings-verktøyene som kan brukes, sammenlignet med andre konvensjonelle datasystemer som mobile-, personlige- og tjeneste-maskiner.
+Innvevde datasystemer blir gradvis en større del av vår hverdag.
+Disse datamaskinene må stadig tilpasse seg nye domener, slik som \emph{tingenes internett}.
```

Comment:
Kanskje det er på sin plass å emphe "Internet of Things" i abstract også, siden det gjøres her?
Jeg tenkte å gjøre det første gangen, men lot være siden det ikke var gjort i abstract....